hazelcast-spring/src/main/resources/hazelcast-spring-3.10.xsd

```diff
@@ -1895,7 +1895,7 @@
             </xs:documentation>
             <xs:appinfo>
                 <tool:annotation>
-                    <tool:exports type="com.hazelcast.core.ReplicatedMap"/>
+                    <tool:exports type="com.hazelcast.replicatedmap.ReplicatedMap"/>
```

Comment:
I think 3.x XSD files should not be updated.

Reply:
I have reverted it. It was automatically converted by Intellij.

Thanks for the review.